### PR TITLE
Adding Cisco CVE-2022-20698

### PR DESCRIPTION
**Pull request by santosomar**

### CODE REVIEW

File: 2022/20xxx/CVE-2022-20698.json [hunk 1]
```diff
@@ -1,18 +1,101 @@
 {
```
Comment (cve-team) [inline 2022/20xxx/CVE-2022-20698.json:1]: 'value' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/20xxx/CVE-2022-20698.json (hunk 1 above)
Comment (santosomar) [reply 2022/20xxx/CVE-2022-20698.json:1]: Fixed typo

File: 2022/20xxx/CVE-2022-20698.json [hunk 2]
```diff
@@ -1,18 +1,104 @@
...
```
Comment (cve-team) [inline 2022/20xxx/CVE-2022-20698.json:1]: 'value' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/20xxx/CVE-2022-20698.json (hunk 2 above)
Comment (santosomar) [reply 2022/20xxx/CVE-2022-20698.json:1]: Fixed in https://github.com/CVEProject/cvelist/pull/4053/commits/27d5b35156045e6253795982d52f4ee296a1d45d

**Review by santosomar — COMMENTED** [review]
Fixed in https://github.com/CVEProject/cvelist/pull/4053/commits/27d5b35156045e6253795982d52f4ee296a1d45d

**Pull request merged**